teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -691,6 +691,8 @@ interface TestOccurrence {
     val newFailure: Boolean
 
     val buildId: BuildId
+    val nextFixed: BuildId
```

Comment:
I think we can invent better names here, we don't need to reuse names from rest API. What about `fixedIn` and `firstFailedIn`?

Reply:
I wanted these names to match the rest parameter names, so it would be obvious at the first sight. Of course, I'll change it, if you don't think it's a good approach.

Reply:
This library is supposed to be easy for use for those who aren't familiar with TeamCity REST API, it isn't just a wrapper around REST queries, see [CONTRIBUTING.md](https://github.com/JetBrains/teamcity-rest-client/blob/master/CONTRIBUTING.md). So if we need to choose between a good name and a name with matches parameter from REST API, we should choose the good name.